Error when running scaffold script

When running `make setup` task after selecting stack services an error is thrown:

```shell
cp ./scaffold/templates/make/docker.mk ./config/docker/
cp: ./scaffold/templates/make/docker.mk: No such file or directory
make[1]: *** [_docker] Error 1
```

This is due a typo in `docker.mk` source path, should be `/scaffold/make` but is set to `/scaffold/templates/make`

--- a/scaffold/make/setup.mk
+++ b/scaffold/make/setup.mk
@@ -68,7 +68,7 @@ _docker:
 	cp -r ./scaffold/templates/docker/.ssl ./
 	mkdir -p ./config/docker
 	cp ./scaffold/templates/docker/docker-compose.*.yml ./config/docker/
-	cp ./scaffold/templates/make/docker.mk ./config/docker/
+	cp ./scaffold/make/docker.mk ./config/docker/
 
 _setup_drupal:
 	@echo "Setup Drupal files";

